connectivity/drivers/ble/FEATURE_BLE/COMPONENT_CYW43XXX/HCIDriver.cpp

```diff
@@ -103,9 +103,6 @@ class HCIDriver : public CordioHCIDriver {
     {
         //Prevent PSoC6 to enter deep-sleep till BT initialization is complete
         sleep_manager_lock_deep_sleep();
-        rtos::ThisThread::sleep_for(500ms);
-        bt_power = 1;
```

Comment:
Is it necessary to keep `bt_power` member variable ? It doesn't seems to be used anymore (except for initialization). 
I also noticed that it is initialized differently in the transport (PullNone) from the HCI (PullUp) driver.

Reply:
Hi @pan,
I updated code according to your comment. Additionally cleaned some indentations (replaced tab to spaces)

Regards,
Nazar